src/cli.ts

```diff
@@ -24,7 +24,9 @@ import reset from './commands/reset';
 
 import ConsoleLogger from './logger';
 
-const shepherdDir = path.join(homedir(), '.shepherd');
+const { DOT_SHEPHERD_DIRECTORY_LOCATION } = process.env;
```

Comment:
TravisCI needs to run with env var set.

Reply:
Nice! Nit: consider just `DOT_SHEPHERD_DIRECTORY` - `LOCATION` feels redundant to me.